migration-engine/cli/tests/cli_tests.rs

```diff
@@ -355,6 +356,147 @@ fn introspect_empty_database() {
     assert!(response.starts_with(r##"{"jsonrpc":"2.0","error":{"code":4466,"message":"An error happened. Check the data field for details.","data":{"is_panic":false,"message":"The introspected database was empty.","meta":null,"error_code":"P4001"}},"id":1}"##));
 }
 
+// TODO: it works fine if run once, it fails with a "Relation already exists" error if run twice in a row.
```

Comment:
Try with something simpler. Do no modifications to the database, just execute a simple statement. We don't test any schema changes anyhow here, just that we can execute a query.